health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -439,33 +437,42 @@ const FormulaConfiguration = ({ onSelect, category, customProps, formulas: initi
 
         {formulasPopUP && (
           <PopUp
-            className={"popUpClass"}
+            className={"popUpClass new-assumption-pop"}
             type={"default"}
             heading={t("CONFIRM_NEW_FORMULA")}
             equalWidthButtons={true}
             children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)